esmvaltool/diag_scripts/ocean/diagnostic_model_vs_obs.py

```diff
@@ -63,7 +63,7 @@
 logging.getLogger().addHandler(logging.StreamHandler(sys.stdout))
 
 
-def add_map_subplot(subplot, cube, nspace, title='', cmap='', log=False):
+def add_map_subplot(subplot, cube, nspace, title='', cmap='', extend='neither', log=False):
```

Comment:
If you add new a new argument to a method, make sure that you also add the new argument to the documentation for this function too. Thanks!

Reply:
@ledm I updated the function description as You suggested! 
I also took a bit of time to update the documentation in [recipes/recipe_oceans.rst](https://github.com/ESMValGroup/ESMValTool/blob/version2_CRESCENDO_WP5_ocean/doc/sphinx/source/recipes/recipe_oceans.rst).